invenio_curations/requests/curation.py

```diff
@@ -185,11 +196,11 @@ def execute(self, identity, uow):
 class CurationRequest(RequestType):
     """Curation request type."""
 
-    type_id = "rdm-curation"
-    name = _("Curation")
+    type_id: ClassVar[str] = "rdm-curation"
+    name: ClassVar[str] = _("Curation")
 
-    create_action = "create"
-    available_actions = {
+    # Dict mapping action names to action classes
+    available_actions: ClassVar[dict[str, Any]] = {
```

Comment:
i think here it would also be possible to add the base class:

https://github.com/inveniosoftware/invenio-requests/blob/master/invenio_requests/customizations/actions.py#L15